<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>554</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Регистрация участников</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QWidget" name="team_1_widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWidget" name="team_1_name_widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="team_1_name_label">
              <property name="text">
               <string>Команда 1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="team_1_name_lineEdit"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="team_1_table_widget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QTableWidget" name="team_1_tableWidget"/>
            </item>
            <item>
             <widget class="QLabel" name="team_1_new_member_label">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Новый участник</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="team_1_new_member_lineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="team_1_button_widget" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QPushButton" name="team_1_add_member_pushButton">
                 <property name="text">
                  <string>Добавить</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="team_1_del_member_pushButton">
                 <property name="text">
                  <string>Удалить</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>60</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="team_2_widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QWidget" name="team_2_name_widget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="team_2_name_label">
              <property name="text">
               <string>Команда 2</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="team_2_name_lineEdit"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="team_2_table_widget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QTableWidget" name="team_2_tableWidget"/>
            </item>
            <item>
             <widget class="QLabel" name="team_2_new_member_label">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Новый участник</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="team_2_new_member_lineEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="team_2_button_widget" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QPushButton" name="team_2_add_member_pushButton">
                 <property name="text">
                  <string>Добавить</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="team_2_del_member_pushButton">
                 <property name="text">
                  <string>Удалить</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="action_Accepted">
   <property name="text">
    <string>ОК</string>
   </property>
   <property name="toolTip">
    <string>ОК</string>
   </property>
  </action>
  <action name="action_Rejected">
   <property name="text">
    <string>Отмена</string>
   </property>
   <property name="toolTip">
    <string>Отмена</string>
   </property>
  </action>
  <action name="action_team_1_add_member">
   <property name="text">
    <string>Добавить участника в команду 1</string>
   </property>
   <property name="toolTip">
    <string>Добавить участника в команду 1</string>
   </property>
  </action>
  <action name="action_team_2_add_member">
   <property name="text">
    <string>Добавить участника в команду 2</string>
   </property>
   <property name="toolTip">
    <string>Добавить участника в команду 2</string>
   </property>
  </action>
  <action name="action_team_1_del_member">
   <property name="text">
    <string>Удалить участника из команды 1</string>
   </property>
   <property name="toolTip">
    <string>Удалить участника из команды 1</string>
   </property>
  </action>
  <action name="action_team_2_del_member">
   <property name="text">
    <string>Удалить участника из команды 2</string>
   </property>
   <property name="toolTip">
    <string>Удалить участника из команды 2</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>team_1_add_member_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>action_team_1_add_member</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>team_2_add_member_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>action_team_2_add_member</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>team_1_del_member_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>action_team_1_del_member</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>team_2_del_member_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>action_team_2_del_member</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>614</x>
     <y>465</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
